ocaml/xapi/xapi_xenops.ml

```diff
@@ -667,10 +667,12 @@ module MD = struct
                   |> List.assoc Xapi_globs.vgpu_type_id in
     let uuid = vgpu.Db_actions.vGPU_uuid in
     let extra_args = vgpu.Db_actions.vGPU_extra_args in
+    let config_file = Db.VGPU_type.get_internal_config ~__context ~self:vgpu_type
```

Comment:
Let's call `Db.VGPU_type.get_internal_config` just once (it's a slave->master call each time). It's already done in L666.